tox.ini

```diff
@@ -25,9 +25,5 @@ deps = -rtest_requirements.txt
 commands =
     python -m pre_commit run {posargs:--all}
 deps = pre-commit>=1.17.0
-extras =
 skip_install = true
 usedevelop = false
-
-[testenv:cov-report]
```

Comment:
Is this intentionally removed?
It's referenced in the [contributing docs](https://github.com/cookiecutter/cookiecutter/pull/1216/files#diff-6a3371457528722a734f3c51d9238c13R117)

Reply:
Removed intentionally.
This can confuse user who do not read contr guide or does not remember this command from it. 
If your run tox - better run with tests. 
Pytest show report after each run.

Reply:
Over it cov-report not comparing anything, it shows only coverage as it is.
So this is confusing. I will rewrite this part.

Reply:
Restored with comments and docs change